ext/strings.go

```diff
@@ -1062,6 +934,49 @@ func makeMatcher(locale string) (language.Matcher, error) {
 	return language.NewMatcher(tags), nil
 }
 
+type callbackImpl struct {
```

Comment:
nit: The idiomatic way to state that a struct implemented a interface is `var _ typeName = InterfaceName{}`. Typically the struct is given it's own descriptive name. Maybe `stringFormatter` ?

Reply:
Makes sense, I've renamed it to `stringFormatter`.